starwars/src/index.js

```diff
@@ -1,7 +1,4 @@
-import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
-import 'bootstrap/dist/css/bootstrap.css';
-
-ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
There is no render function because it was deleted, so nothing is being rendered at all.